### PR TITLE
Price conversion to ref value

### LINKED ISSUE #29: InfluxDB query issue
Panic when invoking get prices/balances api with more than one market_ids.

### PR DESCRIPTION
**Pull request by sekulicd**
This adds price conversion to reference value.

This closes #28 
This closes #29 

@tiero please review.

### CODE REVIEW

**Comment by tiero** [comment]
1. We should use CoinGecko
2. We should map out all known asset hashes to his CoinGecko ID

Ideally when used at scale with coins that are not listed on CoinGecko, we would always convert base to quote (USDT most likely or other well known stablecoins) 
I do expect only Bitcoin in short term to be knonw mapping asset hash -> CoinGecko ID

File: internal/core/application/market_price_service.go [hunk 1]
```diff
@@ -112,6 +145,124 @@ func (m *marketPriceService) GetPrices(
...
+	referenceCurrency string,
+	baseAsset string,
+	quoteAsset string,
+	refPricesPerAssetPair map[string]struct {
+		basePriceInRefCurrency  decimal.Decimal
+		quotePriceInRefCurrency decimal.Decimal
+	},
+	quotePrice decimal.Decimal,
+) (decimal.Decimal, decimal.Decimal, error) {
+	var basePriceInRefCurrency, quotePriceInRefCurrency decimal.Decimal
```
Comment (tiero) [inline internal/core/application/market_price_service.go:159]: these will remain 0 if not convertable?

File: test/application/run_suite_test.go [hunk 2]
```diff
@@ -5,6 +5,6 @@ import (
 	"testing"
 )
 
-func TestInfluxDBTestSuite(t *testing.T) {
+func TestAppSvcTestSuit(t *testing.T) {
```
Comment (tiero) [inline test/application/run_suite_test.go:8]: typo?

```suggestion
func TestAppSvcTestSuite(t *testing.T) {
```

File: internal/core/application/market_price_service.go (hunk 1 above)
Comment (sekulicd) [reply internal/core/application/market_price_service.go:159]: yes

File: test/application/run_suite_test.go (hunk 2 above)
Comment (sekulicd) [reply test/application/run_suite_test.go]: Done

**Review by tiero — APPROVED** [review]
utACK

**Pull request merged**